library/std/src/os/mod.rs

```diff
@@ -17,6 +17,9 @@ cfg_if::cfg_if! {
         #[stable(feature = "rust1", since = "1.0.0")]
         pub use crate::sys::windows_ext as windows;
 
+        #[cfg(not(target_os = "wasi"))]
+        pub use crate::sys::wasi_ext as wasi;
```

Comment:
Does this not need a `stable`/`unstable` attribute?

Reply:
I don't know. The existing `linux` module doesn't have one, so I assumed WASI didn't either, but I'd be happy to add one otherwise.

Reply:
Maybe it's not necessary, but I saw it on the `windows` and `unix` ones, so I thought I'd check.

Reply:
The linux module has a stability attribute, just inside the module itself 

https://github.com/rust-lang/rust/blob/019610754363d1d92a8d0f364d2c0909d6f53dfd/library/std/src/os/linux/mod.rs#L3

I would think this requires a stability attribute, but I'm not sure because the tests didn't complain.